docs/integrations/adding-your-own-connectors.md

```diff
@@ -0,0 +1,49 @@
+# Adding your own connectors
```

Comment:
Should this be renamed `Custom Connectors`? It's less confusing if we can standardize how we refer to features.

Reply:
`Custom Connectors` is way better indeed. Will make the change!

Reply:
Committed the changes